scripts/resultsSum.pl

```diff
@@ -135,10 +134,9 @@ sub resultReporter {
 						if (($diagnostic eq 'failure') || ($diagnostic eq 'all')) {
 							$tapString .= $output;
 						}
-						if ($spec =~ /zos/) {
-							my $dmpDir = dirname($resultFile).'/'.$testName;
-							moveTDUMPS($output, $dmpDir);
-						}
+						require "moveDmp.pl";
+						my $dmpDir = dirname($resultFile).'/'.$testName;
+						moveTDUMPS($output, $dmpDir, $spec);
```

Comment:
we should keep `if` in resultsSum.pl. During test execution, we only need to take care of zos.

Reply:
Hey Lan! Thanks for your feedback! The change was made. @llxia